packages/server/lib/gui/events.ts

```diff
@@ -529,7 +529,7 @@ module.exports = {
         initializeProject (args: LaunchArgs, options: OpenProjectLaunchOptions, browsers: FoundBrowser[]) {
           return openProject.create(args.projectRoot, args, options, browsers)
         },
-        insertProject () {
+        insertProject (projectRoot: string) {
```

Comment:
Do we need this? It's not implemented here - I'm not sure we need this, I think the goal is eventually to not define anything here.

Reply:
The `// TODO` is where I was imagining we'd call the server code that does `insertProject` from cache, so we're "injecting" the untyped code using a type-safe API rather than directly importing it